lib/core/src/server/dev-server.js

```diff
@@ -63,11 +68,19 @@ export default async function(options) {
   }
 
   const iframeStartTime = process.hrtime();
+  let iframeTotalTime;
   const iframeCompiler = webpack(iframeConfig);
   const devMiddlewareOptions = {
-    noInfo: true,
     publicPath: iframeConfig.output.publicPath,
-    watchOptions: iframeConfig.watchOptions || {},
+    watchOptions: {
+      aggregateTimeout: 1,
```

Comment:
What's the reason behind setting this to 1?

Webpack itself sets it to 300 per default
https://webpack.js.org/configuration/watch/#watchoptions-aggregatetimeout

Reply:
get faster HMR